cypress/e2e/page/annotation-pagetests.cy.js

```diff
@@ -38,11 +38,17 @@ describe("tests on annotation page ui with programmatic state loading and store
                 cy.window().its("$nuxt.$router").then(router => {
 
                     router.push({ path: "/annotation" });
+                    cy.commitToVuexStore("setCurrentPage", "annotation");
```

Comment:
Do we have to set `setCurrentPage` explicitly here because we're navigating to it in an atypical way for "cypress reasons"? i.e. we wouldn't have to do this during normal operations because some other handler already does the `setCurrentPage` setting for us automatically?

If so, then I think this line needs a comment that explains this fact. If the NOTE above refers to this, then I may have missed it.

Reply:
Yea, I believe it's due to Cypress reasons. I took inspiration from the `categorization-pagetests.cy.js`. I'll add a note there for clarification.